scripts/plugins/deploy_plugins.py

```diff
@@ -270,28 +270,60 @@ def Deploy_Plugins():
                 )
 
                 try:
-                    result = subprocess.run(
-                        [
-                            plugin_deploy_language,
-                            plugin_deploy_script_path,
-                            stack_action,
-                        ],
-                        universal_newlines=True,
-                        capture_output=True,
+                    # error_file = "stderr_{}.log".format(plugin_name)
+                    # with open(error_file,"wb") as err:
+                    process = subprocess.Popen([
+                        plugin_deploy_language,
+                        plugin_deploy_script_path,
+                        stack_action,
+                    ],
+                    # stdout=subprocess.PIPE,
+                    # TODO: adding any form of stderr to the above call causes the below polling to not work properly for real time output
+                    #       instead, it causes the script to wait until everything is complete before outputting anything
+                    #       tried: stderr=err, stderr=subprocess.PIPE, stderr=subprocess.STDOUT
+                    #       for stderr=err, err is a file: with open(error_file,"wb") as err: 
+                    # stderr=err,
+                    # stderr=subprocess.STDOUT,
+
+                    # TODO: oddly, if you set only stderr=subprocess.PIPE and NOT stdout, 
+                    #       the process.poll() below being focused on stderr does seem to output both stdout and stderr in real time
+                    stderr=subprocess.PIPE,
+                    universal_newlines=True
```

Comment:
This was strange to me..  If we do both `stderr` and `stdout` like: 
```
stdout=subprocess.PIPE,
stderr=subprocess.PIPE
```
..the below polling doesn't seem to work.
even if stderr is one of:
- `err` (i.e. a file opened with `with open(error_file, "wb") as err:
- `subprocess.PIPE`
- `subprocess.STDOUT`

..then the weird thing is that if we do just `stdout=subprocess.PIPE,`, we do not get errors, but if we do `stderr=subprocess.PIPE`, we get both stdout and stderr real time output 🤷

Reply:
Guessing it's just combining two streams into one if we don't need to process these streams separately.

I see where it comes from. Based on https://docs.python.org/3/library/subprocess.html#using-the-subprocess-module

> If capture_output is true, stdout and stderr will be captured. When used, the internal [Popen](https://docs.python.org/3/library/subprocess.html#subprocess.Popen) object is automatically created with stdout=PIPE and stderr=PIPE. The stdout and stderr arguments may not be supplied at the same time as capture_output. If you wish to capture and combine both streams into one, use stdout=PIPE and stderr=STDOUT instead of capture_output.

Can you try the following?
```py
process = subprocess.Popen(command, stdout=subprocess.PIPE, stderr=subprocess.STDOUT)
for combined_output in process.stdout:
    print(combined_output)
```
It worked for me in quick console tests.
That would be at least more intuitive and follow the official docs.
If works, we can reference that documentation reference then.

Reply:
@armab https://github.com/bitovi/bitops/pull/337#pullrequestreview-1158295846

Reply:
also, the quote from the link above is for `subprocess.run` not `subprocess.Popen`